hack/verify-generated-crd.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -euo pipefail
+
+TMP_DIR="$(mktemp -d)"
```

Comment:
Should we clean up `$TMP_DIR`?

Reply:
I wanted to see the stuff when debugging... temp is ephemeral and the data generated is tiny, so not sure. Could make it configurable?